apps/web/app/api/admin/merchants/[id]/route.ts

```
@@ -12,12 +12,26 @@ export async function PATCH(
     const { id } = await params;
-    const { action } = await request.json();
+    const { action, reason } = await request.json();
+    
+    const auditContext = {
+      adminUserId: auth.userId,
+      ipAddress: auth.ipAddress,
+      userAgent: auth.userAgent,
+      reason,
+    };
+
     let data;
```

Biome (2.4.9)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)